.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: C
+
+install:
+  - CHANNEL=http://eupsforge.net/conda/dev ./setup/install.sh lsst-sims
```

Comment:
Removing the double quotes around `$PACKAGE` at [line 59](https://github.com/jchiang87/Twinkles/blob/u/jchiang/connecting_to_travis-ci/setup/install.sh#L59) of `install.sh` fixed the problem.